### DISCUSSION TITLE
umi4 如何查看 server 里的 files

### DISCUSSION THREAD

**Discussion by stefango** [umijs/umi#11334] [Q&A]
有类似下面的功能不？有些 css 文件 404，想看看全量的文件是啥
> https://webpack.js.org/configuration/dev-server/
> If you're having trouble, navigating to the /webpack-dev-server route will show where files are served. For example, http://localhost:9000/webpack-dev-server.

**Comment by fz6m** [comment]
[devServer 选项怎么配置](https://umijs.org/docs/introduce/faq#devserver-%E9%80%89%E9%A1%B9%E6%80%8E%E4%B9%88%E9%85%8D%E7%BD%AE)
